azure-container-apps/ADVANCED.md

```diff
@@ -0,0 +1,321 @@
+| Very high | >5,000          | 1.0   | 1.0Gi  |
+
+If you're provisioning more than 1,000 users, update the resources assigned to the SCIM bridge container to follow these recommendations. The resources specified for the Redis container don't need to be adjusted.
+
+> [!TIP]
+> Learn more about [Container App Name (`ConAppName`) variable requirements](#container-app-name-requirements) that are referenced in the commands below. Copy the following command to a text editor and replace `$ConAppName` and `$ResourceGroup` with the names from your deployment similar to how you did originally in our [deployment guide](https://support.1password.com/scim-deploy-azure/#22-define-variables). 
+
+### Default deployment
+
+If you're provisioning up to 1,000 users, run the following command to reset the specs back to the default:
+
+```bash
+az containerapp update -n $ConAppName -g $ResourceGroup --container-name op-scim-bridge --cpu 0.25 --memory 0.5Gi
+```
+
+To udpate back to the defaults within the Azure Portal: 
+
+1. Within Container App from the [Azure Container Apps Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps), select **Containers** from the sidebar.
+2. Click **Edit and deploy**.
+3. Select the checkbox next to your **op-scim-bridge** container, then choose **Edit**.
+4. Set the **CPU cores** to `0.25` and the **Memory (Gi)** to `0.5`.
+5. Click **Save**, then click **Create**.
+
+### High-volume deployment
+
+If you're provisioning between 1,000 and 5,000 users, run the following command:
+
+```bash
+az containerapp update -n $ConAppName -g $ResourceGroup --container-name op-scim-bridge --cpu 0.5 --memory 1.0Gi
+```
+
+To udpate to the high-volume within the Azure Portal: 
+
+1. Within Container App from the [Azure Container Apps Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps), select **Containers** from the sidebar.
+2. Click **Edit and deploy**.
+3. Select the checkbox next to your **op-scim-bridge** container, then choose **Edit**.
+4. Set the **CPU cores** to `0.5` and the **Memory (Gi)** to `1.0`.
+5. Click **Save**, then click **Create**.
+
+### Very high-volume deployment
+
+If you're provisioning more than 5,000 users, run the following command:
+
+```bash
+az containerapp update -n $ConAppName -g $ResourceGroup --container-name op-scim-bridge --cpu 1.0 --memory 1.0Gi
+```
+
+To udpate to the very high-volume within the Azure Portal: 
+
+1. Within Container App from the [Azure Container Apps Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps), select **Containers** from the sidebar.
+2. Click **Edit and deploy**.
+3. Select the checkbox next to your **op-scim-bridge** container, then choose **Edit**.
+4. Set the **CPU cores** to `1.0` and the **Memory (Gi)** to `1.0`.
+5. Click **Save**, then click **Create**.
+
+## Customize your deployment
```

Comment:
There are a lot of sub-headers here. Could we make them all <details> elements so they're collapsible and the article isn't as long to skim? The previous section may also be nicer with this.